File: Sokoban_2.0_Console_Tester/Sokoban_2.0_Console_Tester/Control/Event.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Sokoban_2._0_Console_Tester
{
    public class Event
    {
        private string keyPressed;
        private double liquid = 0;

        public Event(string keyPressed)
        {
            this.keyPressed = keyPressed;
        }

        public Event(string keyPressed, double liquid)
        {
            this.keyPressed = keyPressed;
            this.liquid = liquid;
        }

        public double GetLiquid()
        {
            return liquid;
        }

        public string GetKeyPressed()
        {
            return keyPressed;
        }
    }
}

File: sokoban/src/hu/bme/sch/sokoban/Event.java
package hu.bme.sch.sokoban;

public class Event {
    private String keyPressed;
    private double liquid = 0;

    public Event(String keyPressed) {
        this.keyPressed = keyPressed;
    }

    public Event(String keyPressed, double liquid) {
        this.keyPressed = keyPressed;
        this.liquid = liquid;
    }

    public double getLiquid() {
        return liquid;
    }

    public String getKeyPressed() {
        return keyPressed;
    }
}
